docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: "2"
+services:
+  db:
+    image: postgres
```

Comment:
This is equivalent to `postgres:latest` which is currently v10.1 but will change over time. I'd recommend pinning to a major version:
https://hub.docker.com/_/postgres/

Also the alpine image is only 14MB vs the 103MB of the standard image - so possibly worth trying that for faster CI and local development bootstrap times.